public/scripts/config.js

```
@@ -460,106 +460,348 @@ document.addEventListener("keypress", (ev) => {
+    case "error":
+      savePopupInstance.error(
+        "Save Failed",
+        "Unable to save configuration. Please check your connection and try again."
+      );
+      break;
+  }
+}
 
-  if (!response.ok) throw new Error("Server response was not ok");
-  alert("Module saved!");
+function showValidationError() {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'showValidationError'. Consider to delete it or rename it.

'showValidationError' is defined here:


(lint/suspicious/noRedeclare)